addons/website_forum/tests/test_forum.py

```diff
@@ -198,10 +198,36 @@ def test_vote_crash(self):
 
     def test_vote(self):
         self.post.create_uid.karma = KARMA['ask']
-        self.user_portal.karma = KARMA['upv']
-        self.post.with_user(self.user_portal).vote(upvote=True)
+        self.user_portal.karma = KARMA['dwv']
+        initial_vote_count = self.post.vote_count
+        res = self.post.with_user(self.user_portal).vote(upvote=True)
+
+        self.assertEqual('1', res['user_vote'])
```

Comment:
I prefer to use the form "assertEqual(value, expected)" ... seems more natural when reading asserts. Especially when expected is an integer :D (I expect user_vote to be 1, not 1 to be user_vote).

Reply:
Touchy subject (see on [SO](https://stackoverflow.com/questions/2404978/why-are-assertequals-parameters-in-the-order-expected-actual])), but I'll comply with your request ;-)

Reply:
The internet that does not agree with me is wrong.